django-stubs/views/decorators/debug.pyi

```diff
@@ -1,9 +1,9 @@
 from collections.abc import Callable
-from typing import Any, Literal, TypeVar
+from typing import Literal
 
-_F = TypeVar("_F", bound=Callable[..., Any])
+from . import _ViewFuncT
 
 coroutine_functions_to_sensitive_variables: dict[int, Literal["__ALL__"] | tuple[str, ...]]
 
-def sensitive_variables(*variables: str) -> Callable[[_F], _F]: ...
-def sensitive_post_parameters(*parameters: str) -> Callable[[_F], _F]: ...
+def sensitive_variables(*variables: str) -> Callable[[_ViewFuncT], _ViewFuncT]: ...
+def sensitive_post_parameters(*parameters: str) -> Callable[[_ViewFuncT], _ViewFuncT]: ...
```

Comment:
`@sensitive_*` decorators can be applied to any regular function, not just view functions.